deployment/stop-monitoring.yaml

```
@@ -10,5 +10,5 @@
   tasks:
 
     - name: Stop monitoring service
-      shell: "killall spacenet-health; sleep 3; killall -9 spacenet-health; true"
+      shell: "killall spacenet-health; sleep 3; killall -9 spacenet-health; true; tmux kill-session -t health"
```

```suggestion
      shell: "killall spacenet-health; sleep 3; killall -9 spacenet-health; tmux kill-session -t health; true"
```
The `true` at the end is to ignore the previous commands potentially failing (e.g. if there is no health service running).